### ISSUE TITLE
Preview php file is not loaded

### ISSUE THREAD

**Issue by gfazioli** [getblocklab/block-lab#208]
Hi, great plugin, but seems that the `blocks/preview-{slug}.php` file is never loaded.

I checked a couple of functions there, for example, `block_lab_template_part()` and the second args `type` is always set to `block` and never on `preview`

Thx in adv

**Comment by gfazioli** [comment]
Humm ok, I found
```php
@param string $type The type of template to load. Only 'block' supported at this stage.
```
so, this means, now `preview` is not supported

**Comment by audetcameron** [comment]
There are docs for preview | https://github.com/getblocklab/block-lab/wiki/Creating-Preview-Templates

Is this not working yet?

**Comment by gfazioli** [comment]
@audetcameron it seems not working atm, `preview-` is never used 😞

**Comment by audetcameron** [comment]
I did a little workaround with admin only css and front end css to show and hide fields.   It worked for now but I really look forward to the preview template taking control.

**Comment by gfazioli** [comment]
fixed in 1.1.1
